### ISSUE TITLE
Red Eclipse v1.5.3 Crashes when using Weapon Fire 2 + Kick at very close range

### ISSUE THREAD

**Issue by cmspencer109** [red-eclipse/base#330]
I have had this issue happen to me many times, in 1.5.1, 1.5.2, and in 1.5.3. It only happens when I am within kick range, and I use my weapon's secondary fire along with a kick at the same time. When I do so, the game will crash right away, without my shot damaging the player at all. The weapons it has crashed on have been Rifle2, Shotgun2, Sword2, and Pistol2, all paired with a kick using Q.

My computer is a 13" 2010 MacBook Pro, with 8GB ram, Intel Core 2 Duo 2.4ghz processor with a NVIDIA GeForce 320M 256 MB graphics card, and I am running Red Eclipse v1.5.3 on OS X Yosemite.

My game name is "Shooter".

**Comment by qreeves** [comment]
I'm using a MacMini with the same specs, does the error go away if you use `depthbits 24`?

**Comment by cmspencer109** [comment]
Well I don't think depthbits had anything to do with it but I realized why it was crashing. I was playing using my touchpad earlier, and whenever I use my touchpad, I map fire1 to the shift key, fire2 to the command key, and crouch to the c key. The reason that my game crashed was because I would fire shotgun2 at someone and then try to kick them, and because I had mapped fire2 to left command, I was using the mac shortcut CMND + Q without realizing it to quit the game. I tried playing with my mouse and it worked perfectly. Sorry for needlessly opening an issue!
